knpu/episode3/abstract-ship.md

```diff
@@ -23,9 +30,15 @@ Over in PhpStorm, when we look at this function now, `Ship` has a Jedi Factor pr
 but `RebelShip` doesn't need that at all. Since `RebelShip` is extending `Ship` it is
 still inheriting that property. While this doesn't hurt anything it is a bit weird to have
 this extra property on our class that we aren't using at all. And this is also true for
-the `isFunctional` method. In `RebelShip` it's always true, but in `Ship` it reads from an `underRepair`
-property, and again that's just not needed in `RebelShip`. The point being, `Ship` comes with
-extra stuff that we are inheriting but not using in `RebelShip`. 
+the `isFunctional` method. In `RebelShip` it's always true:
+
+[[[ code('4bab20796d') ]]]
+
+but in `Ship` it reads from an `underRepair` property, and again that's just not needed in `RebelShip`:
+
+[[[ code('d89d34a269') ]]]
```

Comment:
I took out the highlighting of `__construct()` where `underRepair` is set - I don't think that detail is important enough.